tests/testirisinterface.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import unittest
+
+try:
+    from iris.tests.stock import lat_lon_cube
+except ImportError:
+    raise unittest.SkipTest("Could not import iris, skipping iris interface "
+                            "tests.")
+
+from holoviews.core.data import Dataset
+from holoviews.core.data.iris import coord_to_dimension
+from holoviews.element.comparison import ComparisonTestCase
+
+class TestCube(ComparisonTestCase):
+
```

Comment:
Great! Maybe we can get some people who are familiar with Iris tests to add some more of their own. :-)